src/rocm_docs/rocm_docs_theme/flavors/rocm-docs-home/left-side-menu.jinja

```diff
@@ -1,3 +1,3 @@
 {%
-set main_doc_link = ("ROCm Documentation Home", projects['rocm'])
+set main_doc_link = ("ROCm documentation", projects['rocm'])

```

Comment:
This is intended to point to the homepage

eg: https://rocm.docs.amd.com/en/latest/index.html

Reply:
yes, my intent is to shorten the name and make it sentence case